default.nix

```
@@ -2,4 +2,8 @@
   config = { };
   overlays = [ (import ./overlay.nix) ];
 } }:
-pkgs.mvn2nix
+{
+	mvn2nix = pkgs.mvn2nix;
+
+	buildMavenRepsitory = pkgs.buildMavenRepsitory;
```

```suggestion
  mvn2nix = pkgs.mvn2nix;

  buildMavenRepsitory = pkgs.buildMavenRepsitory;
```

Please use two spaces instead of tabs. (Also elsewhere)

---

This also applies to the generated dependencies Nix expression

---

I ran `nix-fmt` to solve this.